<launch>
  <rosparam command="load" file="$(find mrover)/config/esw.yaml"/>

  <node name="arm_direct_bridge" pkg="mrover" type="simple_bridge">
    <param name="group_name" value="arm_direct"/>
  </node>
  <node name="arm_translator_bridge" pkg="mrover" type="arm_translator_bridge"/>

  <node pkg="mrover" type="gst_websocket_streamer" name="boom_streamer"
        output="screen">
    <param name="dev_node" value="/dev/boomcam"/>
    <param name="port" value="8084"/>
  </node>
</launch>
